dts/common/nordic/nrf54h20.dtsi

```diff
@@ -168,6 +168,20 @@
 				interrupts = <64 NRF_DEFAULT_IRQ_PRIORITY>,
 					     <65 NRF_DEFAULT_IRQ_PRIORITY>;
 			};
+
+			cpuapp_wdt010: watchdog@14000 {
+				compatible = "nordic,nrf-wdt";
+				reg = <0x14000 0x1000>;
+				status = "disabled";
+				interrupts = <20 NRF_DEFAULT_IRQ_PRIORITY>;
+			};
+
+			cpuapp_wdt011: watchdog@15000 {
+				compatible = "nordic,nrf-wdt";
+				reg = <0x15000 0x1000>;
+				status = "disabled";
+				interrupts = <21 NRF_DEFAULT_IRQ_PRIORITY>;
```

Comment:
_The peripheral ID is the interrupt number (IRQ number)
The peripheral ID is given by bits [22:12] of a peripheral address_
Shall I switch IRQ number to 0x15 or can it stay in decimal?

Reply:
The way it is now is consistent with other nodes, so let it stay like this.

Reply:
Remaining nodes have IRQ given in decimal form.